chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/utils/ColorGradient.java

```diff
@@ -314,4 +386,274 @@ private static final ColorGradient viridis() {
         return new ColorGradient("VIRIDIS", stops);
     }
 
+    private static final ColorGradient cet_r2() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'cet_r2' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4880492)